Documentation/cmdref/cilium-agent.md

```diff
@@ -168,6 +168,8 @@ cilium-agent [flags]
       --enable-pmtu-discovery                                     Enable path MTU discovery to send ICMP fragmentation-needed replies to the client
       --enable-policy string                                      Enable policy enforcement (default "default")
       --enable-policy-secrets-sync                                Enables Envoy secret sync for Secrets used in CiliumNetworkPolicy and CiliumClusterwideNetworkPolicy
+      --enable-recorder                                           Enable BPF datapath pcap recorder
```

Comment:
Unrelated?
```suggestion
```

Reply:
I think this showed uP after a merge conflict.

Reply:
Your PR should not need to add anything from a merge conflict, could you remove it?

Reply:
This line should be removed.